src/routes/exception.ts

```diff
@@ -19,32 +19,59 @@
 
 import * as express from 'express';
 import multer from 'multer';
-import exceptionApi, {
+import propertyExceptionApi, {
 	requestContainsFile,
 } from '../exception/property-exceptions/property-exception-api';
 import { wrapAsync } from '../middleware';
 import featureFlags from '../feature-flags';
 import { Request, Response } from 'express';
 import { ExceptionErrorHandler } from '../exception/property-exceptions/error-handling';
+import missingEntityExceptionApi from '../exception/missing-entity-exceptions/api';
 
-// config
-const router = express.Router({ mergeParams: true });
+/**
+ * Property exceptions
+ */
+const propertyExceptionRouter = express.Router({ mergeParams: true });
 const upload = multer({ dest: '/tmp' });
 
-// routes
-router.post('*', upload.single('exceptionFile'), requestContainsFile);
-router.post('/', wrapAsync(exceptionApi.createProgramException));
-router.post('/entity', wrapAsync(exceptionApi.createEntityException));
+// Middleware
+propertyExceptionRouter.use(ExceptionErrorHandler);
+
+// GET
+propertyExceptionRouter.get('/', wrapAsync(propertyExceptionApi.getProgramException));
+propertyExceptionRouter.get('/entity', wrapAsync(propertyExceptionApi.getEntityException));
+
+// POST
+propertyExceptionRouter.post('*', upload.single('exceptionFile'), requestContainsFile);
+propertyExceptionRouter.post('/', wrapAsync(propertyExceptionApi.createProgramException));
+propertyExceptionRouter.post('/entity', wrapAsync(propertyExceptionApi.createEntityException));
+
+// DELETE
+propertyExceptionRouter.delete('/', wrapAsync(propertyExceptionApi.clearProgramException));
+propertyExceptionRouter.delete(
+	'/entity/:entity',
+	wrapAsync(propertyExceptionApi.deleteEntityException),
+);
+
+/**
+ * Missing entity exceptions
+ */
+const missingEntityExceptionRouter = express.Router({ mergeParams: true });
 
-router.get('/', wrapAsync(exceptionApi.getProgramException));
-router.get('/entity', wrapAsync(exceptionApi.getEntityException));
+// GET
+missingEntityExceptionRouter.get('/', (req, resp) => resp.send({ msg: 'not implemented yet...' }));
```

Comment:
Return 500 with message Not Implemented

Reply:
https://github.com/icgc-argo/argo-clinical/pull/1110/commits/b811828b1992ae8706a879be10675dec5f880dfd